src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -102,8 +102,9 @@ private URI checkDockerHostScheme(URI dockerHost) {
     private String checkDockerCertPath(boolean dockerTlsVerify, String dockerCertPath) {
         if (dockerTlsVerify) {
             if (StringUtils.isEmpty(dockerCertPath)) {
-                throw new DockerClientException(
-                        "Enabled TLS verification (DOCKER_TLS_VERIFY=1) but certifate path (DOCKER_CERT_PATH) is not defined.");
+                return dockerCertPath;
+//                throw new DockerClientException(
```

Comment:
Well, we have the mess in configs for creating connections i..e i hade DockerClientCOnfigBuilderForPlugin, then Builder for factory and then the whole builder for two builders...
:atm: i can be satisfied even with additional boolean in DockerClientConfig...